### DISCUSSION TITLE
What is the minimum required set of GH token permissions needed to use GrimoireLab?

### DISCUSSION THREAD

**Discussion by ethanholz** [chaoss/grimoirelab#644] [Q&A] [answered]
This is somewhat similar to #641, but what is the minimum set of required permissions needed by a Personal Access Token to use the GitHub sources listed in the documentation? I haven't been able to find this listed in the documentation.

**Comment by sduenas** [comment, marked as answer]
Sorry for missing this question. The minimum permission for analyzing public repositories is `public_repo`. For private repositories you will new `repo`. See the images below:

- public repositories
![imagen](https://github.com/chaoss/grimoirelab/assets/833352/bed6f10f-250d-47d9-b195-036344afcbab)

- private repositories

![imagen](https://github.com/chaoss/grimoirelab/assets/833352/76abd804-8b9a-45a9-87e1-65c1e3abc79c)

**Reply by ethanholz** [reply]
No worries! I appreciate you getting back to me! Would this be worthwhile to add to the official documentation?
